src/Akeneo/Bundle/ElasticsearchBundle/Client.php

```diff
@@ -57,7 +57,7 @@ public function __construct(ClientBuilder $builder, array $hosts, $indexName)
     public function index($indexType, $id, array $body, Refresh $refresh = null)
     {
         $params = [
-            'index' => $this->indexName,
```

Comment:
We created 2 indices instead of the current 1 index + 2 types. In fact, types are deprecated and will be dropped by ES (cf. https://github.com/elastic/elasticsearch/issues/15613#issuecomment-239435920)